test/shmemx/Makefile.am

```diff
@@ -31,26 +31,10 @@ check_PROGRAMS += \
 	atomic_nbi \
 	put_signal \
 	put_signal_nbi \
-	shmemx_wait_until_all \
-	shmemx_wait_until_any \
-	shmemx_wait_until_some \
-	shmemx_test_all \
-	shmemx_test_any \
-	shmemx_test_some \
```

Comment:
`shmemx_test_some` is another case of using `shmem_p` instead of `shmem_atomic_set`.

(Yup, they all have that problem - my bad).